### ISSUE TITLE
Date field error after loading form again after a submission

### ISSUE THREAD

**Issue by terryupton** [solspace/craft-freeform#379]
**Describe the bug or issue you're experiencing**
I have a form that contains a date field that is setup as follows:
<img width="517" alt="Screenshot 2022-06-24 at 07 30 59" src="https://user-images.githubusercontent.com/5698596/175476397-2b65cff6-50fb-43d1-977b-e4fae38c14b0.png">

The form contains an element id to update a filed using a connection based on this date.
The connection is as follows:
<img width="494" alt="Screenshot 2022-06-24 at 07 31 58" src="https://user-images.githubusercontent.com/5698596/175476536-9d636507-d143-4b3a-8c44-ad28ad86f815.png">

I am passing the element connect id via the form like so:
```twig
 {% set overrideFreeFormValues = {
              fullName: currentUser.name,
              emailAddress: currentUser.email,
              child: child.title
            } %}
            
            {% set formParams = {
              class:'text-navy-600 w-full',
              inputClass: 'focus:border-navy-500 focus:ring focus:ring-navy-300',
              labelClass: 'text-navy-600',
              submitClass: ' bg-navy-500 hover:bg-pink-500 focus:outline-none focus:border-pink-700 focus:ring focus:ring-navy active:bg-pink-700',
              overrideValues: overrideFreeFormValues,
              childId: child.id,
              useRequiredAttribute: true,
              elementId: child.id
            } %}
            
            {% set form = craft.freeform.form("reEnrolment", formParams) %}
            
            {{ form.render() }}
```

One the first load of this form, it works fine and all the fields are loaded and the date is fixed as todays date (based on the settings).

When I submit the form, everything works as expect and the field `Re-enrolled Date` is updated with the date from the form.
However, when I go back to load the form for a second time, I get the following error:

`Solspace\Freeform\Fields\Pro\DatetimeField::getValue(): Return value must be of type string, DateTime returned`

Any ideas what is causing this?

**Comment by kjmartens** [comment]
Sorry for the delay @terryupton. I can duplicate this in my testing as well. I'll check with a developer to see what we can do about this. 🙂

**Comment by kjmartens** [comment]
This should now be resolved in Freeform v3.13.15. 🙂
